comms.py

```diff
@@ -0,0 +1,62 @@
+from mcu import *
+from controls import *
+
+class Communications:
+    def __init__(self, mcuVAR: MCUInterface, controller: controls):
+        self.mcuVAR = mcuVAR
+        self.controller = controls
+
+        MOTOR_CLAW = 0 
+        MOTOR_ONE = 1
+        MOTOR_TWO = 2
+        MOTOR_THREE = 3 
+
+    def read_send(key_pressed):
+        initial_calibration_percent = 100
+        value_multiplier = 2 #2 percent
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_ONE,1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_TWO, 1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_THREE, 1000)
+
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_ONE,motors_move[i][1], 1500)
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_TWO,motors_move[i][1], 1500)
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_THREE,motors_move[i][1], 1500)
+
+        while controls.running:
+            #Different functionality/motor running process for 
+            if key_pressed == "s":
```

Comment:
I would suggest putting the condition (`"s"`), along with the arguments of the function (`MOTOR_ONE`, `-1*initial_calibration_percent`) in a tuple. Do that for each branch and put the tuples in a list. Then, use a for loop to iterate through the list.

Reply:
perhaps a better solution here would be binding the set of second character to the set of motors, and bind the set of first characters, if present, to the set of possible second parameters.